contracts/dao/voting/investors-vesting-vault/src/msg.rs

```diff
@@ -0,0 +1,40 @@
+use cwd_interface::Admin;
+use cwd_macros::{info_query, voting_query};
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, JsonSchema, Debug, Clone)]
+pub struct InstantiateMsg {
+    // Vesting contract address.
+    pub vesting_contract_address: String,
+    // Description contains information that characterizes the vault.
+    pub description: String,
+    // Owner can update all configs including changing the owner. This will generally be a DAO.
+    pub owner: Admin,
+    // Manager can update all configs except changing the owner. This will generally be an operations multisig for a DAO.
+    pub manager: Option<String>,
+}
+
+#[derive(Serialize, Deserialize, JsonSchema, Debug, Clone)]
```

Comment:
please use macro to fully comply with the voting vault interface. example: https://github.com/neutron-org/neutron-dao/blob/main/packages/neutron-lockdrop-vault/src/msg.rs#L25-L47

Reply:
also here's a tip how to define handlers for disabled bonding: https://github.com/neutron-org/neutron-dao/blob/feat/NTRN-380_vesting_vault/contracts/dao/voting/vesting-lp-vault/src/contract.rs#L312